test/e2e/specs/all-features-confirmation/recordset/presentation.spec.js

```diff
@@ -300,6 +301,13 @@ describe('View recordset,', function() {
                         });
                     });
                 });
+
+                //Check tooltip of Action column
+                var actionCol = element(by.cssContainingText('table > thead > tr > th > span', 'Actions'));
```

Comment:
This will work for now, but if we change the structure of the HTML this test case will need to be updated. I think it's better if you add a class to the DOM like the rest of the `th` tags. We can call the class `actions-header` (no reason so complicate it and use actions in one case and view in another.